spidermon/contrib/scrapy/pipelines.py

```diff
@@ -96,6 +97,21 @@ def _load_jsonschema_validator(cls, schema):
             )
         return JSONSchemaValidator(schema)
 
+    @classmethod
+    def _load_cerberus_validator(cls, schema):
+        print("WORKING at loading")
+        if isinstance(schema, six.string_types):
+            schema = get_schema_from(schema)
```

Comment:
`get_schema_from` is a function that comes from `spidermon.contrib.validation.jsonschema.tools`. Maybe you will want to use it to get the schemas for Cerberus too and if so, keeping the function inside `jsonschema` is not a good idea (as it are not JSON schema only related). I would move it to `validation.utils` (see if the tests still pass).